en-US/Players_Guide/03_Creating_a_Character/Level_Advancement/Multiclassing.md

```diff
@@ -0,0 +1,139 @@
+
+###### Spellcasting
+
+Your capacity for spellcasting depends partly on your combined levels in all your spellcasting classes and partly on your individual levels in those classes.
+Once you have the Spellcasting feature from more than one class, use the rules below.
+If you multiclass but have the Spellcasting feature from only one class, you follow the rules as described in that class.
+
+**Spells Known and Prepared.**
+You determine what spells you know and can prepare for each class individually, as if you were a single-classed member of that class.
+If you are a Ranger 4/Wizard 3, for example, you know three 1st-level Ranger spells based on your levels in the Ranger class.
+As 3rd-level Wizard, you know three Wizard cantrips, and your spellbook contains ten Wizard spells, two of which (the two you gained when you reached 3rd level as a Wizard) can be 2nd-level spells.
+If your Intelligence is 16, you can prepare six Wizard spells from your spellbook.
+
+Each spell you know and prepare is associated with one of your classes, and you use the spellcasting ability of that class when you cast the spell.
+Similarly, a spellcasting focus, such as a holy symbol, can be used only for the spells from the class associated with that focus.
+
+**Spell Slots.**
+You determine your available spell slots by adding together all your levels in the Bard, Cleric, Sorcerer, Wizard, Wodewose classes, and half your levels (rounded down) in the Ranger and Vanguard classes.
+Use this total to determine your spell slots by consulting the Multiclass Spellcaster table.
+
+If you have more than one spellcasting class, this table might give you spell slots of a level that is higher than the spells you know or can prepare.
+You can use those slots, but only to cast your lower-level spells.
+If a lower-level spell that you cast, like _Burning Hands_, has an enhanced effect when cast using a higher-level slot, you can use the enhanced effect, even though you don’t have any spells of that higher level.
+
+For example, if you are the aforementioned Ranger 4/Wizard 3, you count as a 5th-level character when determining your spell slots: you have four 1st-level slots, three 2nd-level slots, and two 3rd-level slots.
+However, you don’t know any 3rd-level spells, nor do you know any 2nd-level Ranger spells.
+You can use the spell slots of those levels to cast the spells you do know—and potentially enhance their effects.
+
+**Pact Magic.**
+
+If you have both the Spellcasting class feature and the Pact Magic class feature from the Warlock class, you can use the spell slots you gain from the Pact Magic feature to cast spells you know or have prepared from classes with the Spellcasting class feature, and you can use the spell slots you gain from the Spellcasting class feature to cast Warlock spells you know.
+
+<!-- TODO Add the multiclass spellcaster spell slots per spell level table -->
```

Comment:
There are some open comments here; I can either wait for those to be resolved in the Google Doc, or transfer them to GitHub as well. Do you have a preference?